index.html

```diff
@@ -17,24 +34,60 @@
             <h2>
                 Static pages are the best
             </h2>
-        <nav>
-            <ul>
-                <li>GitHub</li>
-                <li>Twitter</li>
-                <li>About</li>
-                <li>Home</li>
-            </ul>
-        </nav>
-        </center>
+            </header>
+            <nav>
+                <ul>
+                    <li class="tab" data-content="home"><a href=#>Home</a></li>
+                    <li class="tab" data-content="about"><a href=#>About</a></li>
+                    <li><a href="http://www.something.com" target=_blank>Twitter</a></li>
+                    <li><a href="http://www.github.com/suhk" target=_blank>Github</a></li>
+                </ul>
+            </nav>
+            </center>
 
-    </header>
-    <main>
-        
-    </main>
+        <div class="filter">
+            Sort by category:
+            <select id="category">
+                <option value="null">None</option>
+            </select>
+        </div>
+        <div class="filter">
+            Sort by author:
+            <select id="author">
+                <option value="null">None</option>
+            </select>
+        </div>
+    <div class="all">
+        <section id="home">
+            <main>
+                <!-->
+                <article>
+                    <h1></h1>
+                    <h3><em><a class="auth"> </a></em></h3>
+                    <h5></h5>
+                    <div class="body"></div>
+                    <a href=# class="expand"></a>
+                    <hr size="2" color="black">
+                </article>
+                <!-->
```

Comment:
It is confusing and adds clutter to have this commented out code here. If you need it here for some reason, explain in a comment why it is here and commented out. Otherwise don't commit it to your master branch.